.isort.cfg

```diff
@@ -0,0 +1,4 @@
+[settings]
+line_length=120
```

Comment:
If you would use 80 char limit, then you would probably wouldn't go such deep in indention levels :)